src/components/trendingBooks/BookCard.tsx

```diff
@@ -0,0 +1,14 @@
+import Link from '../../../node_modules/next/link';
+
+export default function BookCard(props: {title: any, author: any, id: number}) {
```

Comment:
You could define a prop type or use typed object like: 
```suggestion
export default function BookCard({ title, author, id} : { title: string, author: string, id: number } ) {
```

If we type the data we could avoid `any`. When we build the app for production, the build step is going to error out on `any` usage (which is a good thing in general if we're using typescript)